src/packages/solid/components/Input/Input.styles.ts

```diff
@@ -59,7 +59,8 @@ const input: InputConfig = {
   themeKeys: {
     borderRadius: 'borderRadius',
     color: 'backgroundColor',
-    justifyContent: 'justifyContent'
+    justifyContent: 'justifyContent',
```

Comment:
are we sure justifyContent is setup right here? at first glance im not sure and i just want to make sure

Reply:
ur right. ill move it into the container and out of the inputContainer, thanks